applications/values.yaml

```diff
@@ -13,6 +13,9 @@ applicationVersions:
   backend: &backendVersion master
   git-api: &gitApiVersion master
   launcher: &launcherVersion v1.0.0
+  agnosticv-operator: &agnosticvVersion helm_chart
+  anarchy-operator: &anarchyVersion helm_chart
+  poolboy: &poolboyVersion helm_chart
```

Comment:
Please add these new versions to the `omp-backend` / hemValues.versions below so the version display functionality works as expected.